backend/handlers/createtrippsot.js

```diff
@@ -1,6 +1,7 @@
 const trip = require('../../database/triphelpers.js')
```

Comment:
This file is spelt incorrectly. Also please use camelCase or snake_case for file names.